.gitignore

```diff
@@ -2,3 +2,4 @@ phpunit.xml
 vendor/*
 composer.lock
 Tests/fixtures/app/cache
+.idea
```

Comment:
❌ you should add this in your global gitignore but should not be in this oss project.